code/processes/dqc.q

```diff
@@ -1,23 +1,23 @@
+/ - default parameters
 \d .dqe
 
-configcsv:@[value;`.dqe.configcsv;first .proc.getconfigfile["dqcconfig.csv"]];
-dqcdbdir:@[value;`dqcdbdir;`:dqcdb];
-detailcsv:@[value;`.dqe.detailcsv;first .proc.getconfigfile["dqedetail.csv"]];
-gmttime:@[value;`gmttime;1b];
-partitiontype:@[value;`partitiontype;`date];
-writedownperiod:@[value;`writedownperiod;0D01:00:00];
-.servers.CONNECTIONS:`tickerplant`rdb`hdb`dqe`dqedb // set to all so that it connects to everything
-/.servers.CONNECTIONS:`ALL // set to all so that it connects to everything
-
-/- determine the partition value
-getpartition:@[value;`getpartition;
+configcsv:@[value;`.dqe.configcsv;first .proc.getconfigfile["dqcconfig.csv"]];  // loading up the config csv file
+dqcdbdir:@[value;`dqcdbdir;`:dqcdb];                                            // location of dqcdb database
+detailcsv:@[value;`.dqe.detailcsv;first .proc.getconfigfile["dqedetail.csv"]];  // csv file that contains information regarding dqc checks
+gmttime:@[value;`gmttime;1b];                                                   // define wehter the process is on gmttime or not
+partitiontype:@[value;`partitiontype;`date];                                    // set type of partition (defaults to `date)
+writedownperiod:@[value;`writedownperiod;0D01:00:00];                           // dqc periodically writes down to dqcdb, writedownperiod determines the period between writedowns
+.servers.CONNECTIONS:`tickerplant`rdb`hdb`dqe`dqedb                             // set to connect to tickerplant, rdb, hdb, dqe, and dqedb
+
```

Comment:
Commenting consistency with that below

Reply:
The default parameters are usually commented at the end of line in other torq processes, so I decided to comment in a similar fashion